### PR TITLE
Lessons 11

### PR DESCRIPTION
**Pull request by vtx123**

### CODE REVIEW

File: Lesson11/lesson11_2.cpp [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+enum class SortingDirection { ascending, descending };
+
+bool isSorted(const int* arr, int size, SortingDirection direction){
+    for(int i = 0; i < size-1; i++){
+        if(arr[i] > arr[i + 1] && direction == SortingDirection::ascending){
+            return false;
+        }
+        if(arr[i] <= arr[i + 1] && direction == SortingDirection::descending){
+            return false;
+        }
```
Comment (helen-black) [inline Lesson11/lesson11_2.cpp]: you're checking sorting direction on every loop iteration. Please, check it only once

File: Lesson11/lesson11_3.cpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+const int COL = 4;
+const int ROW = 4;
+
+void passArrV1(int arr_2d[ROW][COL])
+{
+    std::cout << "Passing array v1: " << std::endl;
+        for(int j = COL - 1; j >= 0 ; j--){
+            for(int i = 0; i < ROW; i++){
+            arr_2d[i][j] = 0;
```
Comment (helen-black) [inline Lesson11/lesson11_3.cpp:12]: why are you doing it?

**Review by helen-black — APPROVED** [review]

**Review by helen-black — CHANGES_REQUESTED** [review]
Fix the second task, please

File: Lesson11/lesson11_3.cpp (hunk 2 above)
Comment (vtx123) [reply Lesson11/lesson11_3.cpp:12]: a bit comfortable to read, personally. if i'am understand you correctly.

File: Lesson11/lesson11_2.cpp (hunk 1 above)
Comment (vtx123) [reply Lesson11/lesson11_2.cpp]: fixed

**Pull request merged**